src/OPFS/utils.js

```diff
@@ -137,6 +139,96 @@ export function downloadToOPFS(
   })
 }
 
+/**
+ *
+ * @param {*} originalFilePath
+ * @param {*} commitHash
+ * @param {*} owner
+ * @param {*} repo
+ * @param {*} branch
+ * @return {boolean}
+ */
+export function doesFileExistInOPFS(
+    originalFilePath,
+    commitHash,
+    owner,
+    repo,
+    branch) {
+  assertDefined(originalFilePath, commitHash, owner, repo, branch)
+
+  return new Promise((resolve, reject) => {
```

Comment:
I think the only difference between this and the Promise starting on 204 is the string 'exist' (here) and 'deleted' (there).  Please make a helper and pass the string in instead.